src/main/resources/application.yml

```diff
@@ -0,0 +1,10 @@
+spring:
+  jpa:
+    show-sql: true
+    hibernate:
+      ddl-auto: update
+    datasource:
+      driver-class-name: org.postgresql.Driver
+      url:${POSTGRES_DB_URL}
```

Comment:
It is bad approach. Your application yml doesn't know about docker compose environment variables. Specify url here directly